service/policy/db/migrations/20250407000000_add_non_data_resources.sql

```diff
@@ -0,0 +1,55 @@
+-- +goose Up
+-- +goose StatementBegin
+
+CREATE TABLE IF NOT EXISTS non_data_resource_groups (
+  id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+  name VARCHAR NOT NULL UNIQUE,
+  metadata JSONB,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+COMMENT ON TABLE non_data_resource_groups IS 'Table to store non data resource groups';
+COMMENT ON COLUMN non_data_resource_groups.id IS 'Primary key for the table';
+COMMENT ON COLUMN non_data_resource_groups.name IS 'Name for the non data resource group';
+COMMENT ON COLUMN non_data_resource_groups.metadata IS 'Metadata for the non data resource group (see protos for structure)';
+COMMENT ON COLUMN non_data_resource_groups.created_at IS 'Timestamp when the record was created';
+COMMENT ON COLUMN non_data_resource_groups.updated_at IS 'Timestamp when the record was last updated';
+
+CREATE TABLE IF NOT EXISTS non_data_resource_values (
+  id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+  non_data_resource_group_id UUID NOT NULL REFERENCES non_data_resource_groups(id) ON DELETE CASCADE,
+  value VARCHAR NOT NULL,
```

Comment:
I was thinking about this last night and we should make `value` unique under the `non_data_resource_group_id` so you only have one `network : abc` at a time.

Reply:
Ah right good call, that's the one case I forgot about

Reply:
Done